league/templates/league/results_iframe.html

```diff
@@ -0,0 +1,169 @@
+{% load static wagtailuserbar compress puput_tags  %}
+{% load tz static league_tags bootstrap3 i18n %}
+{% load league_tags %}
+{% get_meijin as meijin %}
+
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
+
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- image for socials -->
+  
+  <title>    
+    {% block title %}{{event}}- {% trans "results" %}{% endblock %}
+  </title>
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+  <script src="https://kit.fontawesome.com/0fe78b4c78.js" crossorigin="anonymous"></script>
+  
+  <style type="text/css"> 
```

Comment:
why not `{% extends "full_width.html" %}` and put this in `{% block extra_css %}`? you'll need to target `body` instead of `.iframe`

Reply:
`full_width` starts with `{% extends "base.html" %}`. So html would contain some elements they don't want like menu, footer, bunch of css and js I think.